syncto/tests/test_authentication.py

```diff
@@ -107,3 +107,89 @@ def test_in_case_decryption_fails_it_should_raise(self):
                     self.request.registry.cache, 'get',
                     return_value=tempered_cache):
                 self.assertRaises(CryptoError, build_sync_client, self.request)
+
+    def test_uses_ttl_from_settings_if_assertion_has_no_expiration(self):
+        # This assertion comes from PyBrowserID tests cases.
+        # https://github.com/mozilla/PyBrowserID
+        assertion = """
+        eyJjZXJ0aWZpY2F0ZXMiOlsiZXlKaGJHY2lPaUpTVXpFeU9DSjkuZXlKcGMzTWlPaUppY
```

Comment:
@rfk It seems to me that this assertion doesn't have the same format as the one Firefox Account is using. Should we support this as well?

```
{"certificates":["eyJhbGciOiJSUzEyOCJ9.eyJpc3MiOiJicm93c2VyaWQub3JnIiwiZXhwIjoxMzIxOTQxOTgzMDUwLCJwdWJsaWMta2V5Ijp7ImFsZ29yaXRobSI6IlJTIiwibiI6IjgxNjkxNTA0NTk0NDUzNDU1ODAxNTkyNDkxMzk2NDE4NzE2MTU0NDM4NDYwODIzMzA1MTIyODcwMDQ4NzM1MDI5MDkzMzkyMDY3MDE1MjA0NDgzMjU1NzYwNDA5NzEyODc3MTk0OTUzMTAwMTQ2NDUzNzM2NjE4NTUzNDcxMjI1ODkyMzM1ODUxMjY5MDAwNDAyMTk2ODE0MDkiLCJlIjoiNjU1MzcifSwicHJpbmNpcGFsIjp7ImVtYWlsIjoicnlhbkByZmsuaWQuYXUifX0.k_hhKX0TBVu2_k3m_nD5NUb_NKp__O-651mBEIwKSYdiNzt0BoVFCDTEnxHPMbBV2Zz94X8--TcUrbtEt1euKWVv3-51T9LAfuzHH_zCBQrUnld1ZWJjA3_9dHPy30g4LIOXe2VZgtOSoks2dQ8d3ok9RQ2P0NDG0u00g7yFz18"],"assertion":"eyJhbGciOiJSUzY0In0.eyJleHAiOjEzMjE5Mjk0NzU4MjksImF1ZCI6Imh0dHA6Ly9teWZhdm9yaXRlYmVlci5vcmcifQ.AhgKd4ys4KqgHbXqCRKxGvYnVaI9L-ohXHy4IPUX9m_24Mg_bU6hdH13S6qgC-oHpiKp_Lizp4FF9TrPc60SEw"}```

```
{"alg":"RS128"}
{"iss":"browserid.org","exp":1321941983050,"public-key":{"algorithm":"RS","n":"8169150459445345580159249139641871615443846082330512287004873502909339206701520448325576040971287719495310014645373661855347122589233585126900040219681409","e":"65537"},"principal":{"email":"ryan@rfk.id.au"}}
```

```
{"alg":"RS64"}
{"exp":1321929475829,"aud":"http://myfavoritebeer.org"}
```